opapi/src/state.ts

```diff
@@ -124,7 +124,9 @@ export type OperationWithBodyProps<
   requestBody: {
     description: string
     schema: SchemaOfType<S>
+    format?: 'binary'
```

Comment:
**question:** what does that mean ? I know its an `@anatine/zod-openapi`, but what will be the effect on the resulting openapi ?

Also, can you use:

```ts
{
  requestBody: { format: "binary" },
  contentType: "application/json"
}
```

or is `{ format: "binary" }` absolutely tied with `{ contentType: "*/*" }` ?

Reply:
`format` is tied to a fields of `type: string`. AFAIK, it has nothing to do with the `contentType`.